lstchain/scripts/lstchain_mc_rfperformance.py

```diff
@@ -111,80 +111,33 @@ def main():
 
     ####PLOT SOME RESULTS#####
 
-    gammas = dl2[dl2.gammaness >= 0.5]
-    protons = dl2[dl2.gammaness < 0.5]
-    gammas.reco_type = 0
-    protons.reco_type = 1
-
-    focal_length = 28 * u.m
-    src_pos_reco = utils.reco_source_position_sky(gammas.x.values * u.m,
```

Comment:
is this now calculated within some of the ctaplot functions? If so, where exactly?

Reply:
the `src_pos_reco` you mean?
No, it just does not make sense to compute here anymore, `reco_alt` and `reco_az` are now computed in `apply_models`